src/mmw/js/src/modeling/templates/controls/conservationPractice.html

```diff
@@ -7,14 +7,14 @@
     <div id="conserve-tools" class="dropdown-menu draw-tools menu-left" role="menu"> <!-- Dropdown -->
         <div class="pad-1"> <!-- Dropdown Content -->
             <ul class="row">
-                {{ utils.thumb('Rain Garden', 'rain_garden') }}
-                {{ utils.thumb('Veg Infil Basin', 'veg_infil_basin') }}
-                {{ utils.thumb('Porous Paving', 'porous_paving') }}
+                {{ utils.thumb(getHumanReadableLabel('rain_garden'), 'rain_garden') }}
```

Comment:
Can we make this even DRYer by refactoring the macro to accept only one parameter, and use the `getHumanReadableLabel` function inside the macro?

```
{{ utils.thumb('rain_garden') }}
```

And use the [set](https://mozilla.github.io/nunjucks/templating.html#set) directive to initialize a `label` value inside the macro:

```
{% macro thumb(value, size=56) %}
{% set label = getHumanReadableLabel(value) %}
    ...
    <label class="dark">{{ label }}</label>
    ...
{% endmacro %}
```